random/between.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
```

Comment:
Please add where applicable.
```suggestion
// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
// This module is browser compatible.
```